config/state.go

```diff
@@ -79,6 +123,7 @@ func (state *MixState) Load() error {
 	f, err := os.Open(state.filename)
 	if err != nil {
 		// If state does not exists, create a default state
+		fmt.Println("WARNING: Using FORMAT value from " + state.formatSource)
```

Comment:
I'd say yeah, but technically it's not an error just informative message since it may have found a good source for the format. /me leaves this one to @rchiossi